src/components/request-form/request-form.js

```diff
@@ -60,33 +61,44 @@ const defaultArgVal = {
   json: '{}',
 };
 
+const height = 550;
+
 const languages = [{ label: codeModes.javascript }, { label: codeModes.json }];
 
-export default class RequestForm extends React.Component {
+export default class RequestForm extends PureComponent {
   static propTypes = {
-    width: PropTypes.number.isRequired,
-    height: PropTypes.number.isRequired,
-    path: PropTypes.string.isRequired,
-    input: PropTypes.string,
-    selectLanguage: PropTypes.func.isRequired,
-    setArgument: PropTypes.func.isRequired,
-    submit: PropTypes.func.isRequired,
+    selectedMethodPath: PropTypes.string.isRequired,
+    content: PropTypes.shape({
+      language: PropTypes.string,
```

Comment:
this part of the shape shouldn't be `required` ?

Reply:
you're right